### PR TITLE
Add author header

### PR DESCRIPTION
**Pull request by mlbrgl**
This PR adds a new "Author" OwidGdocType and associated template.

👉 See [this comment](https://github.com/owid/owid-grapher/pull/3256#discussion_r1506571552) for some of the more structural decisions I made.

- [x] responsive styles

[staging preview](http://staging-site-author-header/admin/gdocs/1VDcqVRkbi7dYI0xI9m3AQmp14xCEy9UukJoD6n3rHtE/preview), [gdoc](https://docs.google.com/document/d/1VDcqVRkbi7dYI0xI9m3AQmp14xCEy9UukJoD6n3rHtE/edit)

Only consider the "🌑 Dark launch" scope from the [tracking issue](https://github.com/owid/owid-grapher/issues/3224)

### CODE REVIEW

**Comment by mlbrgl** [comment]
Current dependencies on/for this PR:
* **#3257** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/3257?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3256** <a href="https://app.graphite.dev/github/pr/owid/owid-grapher/3256?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/3256?utm_source=stack-comment).

File: db/model/Gdoc/GdocAuthor.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            // parsing pipeline. In particular, this gives us round trip testing
+            // and the documenting of blocks in a consistent way, wihch is more
+            // justifiable here compared to the bio block given the wider array
+            // of edge cases for this block.
+            content.socials = parseSocials({
+                type: "socials",
+                value: content.socials,
+            })
+        }
+    }
```
Comment (mlbrgl) [inline db/model/Gdoc/GdocAuthor.ts:51]: @ikesau I'm curious if you have any thoughts on the handling of the bio and socials section of the header:
- bio: just a property with possibly rich text
- socials: a full-blown Archie block outside of the [+body]

**Comment by mlbrgl** [comment]
@danyx23 I'm sending you this for review ahead of responsive styles to parallelize structural discussions with the less problematic, more straightforward mobile styles, for which a design review is usually enough.

**Review by danyx23 — APPROVED** [review]
LGTM!

**Comment by mlbrgl** [comment]
### Merge activity

* **Mar 15, 4:42 AM EDT**: @mlbrgl started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/owid/owid-grapher/3256).

**Pull request merged**